tools/shared/utils/symacc.c

```diff
@@ -36,11 +36,11 @@
 #endif
 
 #ifdef UTILSYMTAB
-STB stb;
+extern STB stb;
 GBL gbl;
 #else
 extern STB stb;
-extern GBL gbl;
+GBL gbl;
```

Comment:
These `gbl`s are different from the one I touched in this PR. Removing them causes an "undefined reference" error. But when they are in the code compiles, so they can't be overlapping with the `gbl` which I modified as that would cause a "multiple definition error" we have seen earlier.
Apparently - there are multiple structures, which are declared separately, perhaps on purpose. They have to stay as they are...